src/Reader/Extension/PodcastIndex/Entry.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-feed for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-feed/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-feed/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\Feed\Reader\Extension\PodcastIndex;
+
+use Laminas\Feed\Reader\Extension;
+use stdClass;
+
+class Entry extends Extension\AbstractEntry
+{
+    /**
+     * Get the entry transcript
+     *
+     * @return object
+     */
+    public function getTranscript()
```

Comment:
The possible return values for this method are `stdClass` and `null`. This can be added to the method signature and the DocBlock can be omitted for that.

```suggestion
    /**
     * Get the entry transcript
     */
    public function getTranscript(): ?stdClass
```